blocks/property-listings-cards/property-listings-cards.js

```diff
@@ -0,0 +1,80 @@
+
+function createCard(listing) {
+  let html = `
+    <li class="listing-tile"> 
+      <div class="listing-image-container"> 
+        <div class="property-image"> 
+          <a href="${listing.PdpPath}" rel="noopener" aria-label="${listing.StreetName}"> 
+            <img src="${listing.SmallMedia[0].mediaUrl} alt="property-image" loading="lazy" class="property-thumbnail">
+          </a> 
+        </div>
+        <div class="image-position-top">
+        </div>
+        <div class="image-position-bottom"> 
+          <div class="property-labels">`;
+  if (listing.FeaturedListing) {
+    html += '<span class="property-label featured-listing">Featured Listing</span>';
+  }
+  if (listing.newListing) {
+    html += '<span class="property-label new-listing">NEW LISTING</span>';
+  }
+  html += `
+        </div> 
+          <div class="property-info-wrapper"> 
+            <div class="property-price">
+              ${listing.ListPriceUS}
+            </div> 
+            <div class="property-info"> 
+              <div class="address"> 
+                ${listing.StreetName}
+                <br> 
+                ${listing.City}, ${listing.StateOrProvince} ${listing.PostalCode} 
+              </div> 
+              <div class="specs"> 
+                ${listing.BedroomsTotal} Bed / ${listing.BathroomsTotal} Bath / ${listing.LivingArea} ${listing.LivingAreaUnits}
+              </div> 
+            </div> 
+          </div> 
+        </div> 
+      </div>
+      <div class="property-buttons"> 
+        <div class="buttons-row-flex"> 
+          <a aria-label="Contact Form" href="#" class="button-property"> 
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Pro 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. --><path d="M64 112c-8.8 0-16 7.2-16 16v22.1L220.5 291.7c20.7 17 50.4 17 71.1 0L464 150.1V128c0-8.8-7.2-16-16-16H64zM48 212.2V384c0 8.8 7.2 16 16 16H448c8.8 0 16-7.2 16-16V212.2L322 328.8c-38.4 31.5-93.7 31.5-132 0L48 212.2zM0 128C0 92.7 28.7 64 64 64H448c35.3 0 64 28.7 64 64V384c0 35.3-28.7 64-64 64H64c-35.3 0-64-28.7-64-64V128z"/></svg> 
```

Comment:
use images from the common moves website instead of w3.  see https://github.com/hlxsites/hsf-commonmoves/tree/main/icons